# RNA-Seq
## RNA-seq experimental design
* [a blog post about RNA Seq](http://michelebusby.tumblr.com/post/26913184737/thinking-about-designing-rna-seq-experiments-to)
* [A comprehensive evaluation of normalization methods](https://academic.oup.com/bib/article/14/6/671/189645)
* [A comprehensive evaluation of ensembl, RefSeq, and UCSC annotations in the context of RNA-seq read mapping and gene quantification](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-1308-8)
* [Union Exon Based Approach for RNA-Seq Gene Quantification: To Be or Not to Be?](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0141910)
* [paper: The impact of amplification on differential expression analyses by RNA-seq](https://www.nature.com/articles/srep25533) .Computational removal of read duplicates is not recommended for differential expression analysis.
* [spike ins and normalization based on them](https://www.ncbi.nlm.nih.gov/pubmed/25150836)
## Batch effects
* [TACKLING BATCH EFFECTS AND BIAS IN TRANSCRIPT EXPRESSION](https://mikelove.github.io/eurobioc2015/#/slide-1)
* [Tackling the widespread and critical impact of batch effects in high-throughput data](https://www.nature.com/articles/nrg2825)
